imagebuildah/build.go

```diff
@@ -325,6 +325,9 @@ func (b *Executor) Preserve(path string) error {
 		archivedPath := filepath.Join(b.mountPoint, cachedPath)
 		logrus.Debugf("no longer need cache of %q in %q", archivedPath, b.volumeCache[cachedPath])
 		if err := os.Remove(b.volumeCache[cachedPath]); err != nil {
+			if strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
if err == syscall.ENOENT 
Would be much better.

Reply:
Doesn't seem to work, but this does:

                        if os.IsNotExist(err) {

and I've converted to it.